ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```diff
@@ -52,7 +52,7 @@ ref:
     default: ci.azure.devcluster.openshift.com
```

Comment:
If you buy [this argument][1], you'll want to drop the default here.

[1]: https://github.com/openshift/release/pull/69970#discussion_r2403400956